pinot-core/src/main/java/org/apache/pinot/core/geospatial/transform/function/ScalarFunctions.java

```diff
@@ -98,4 +99,16 @@ public static String stAsText(byte[] bytes) {
     GeometryUtils.setGeometry(geometry);
     return GeometrySerializer.serialize(geometry);
   }
+
+  /**
+   * Gets the H3 hexagon address from the location
+   * @param latitude latitude of the location
+   * @param longitude longitude of the location
+   * @param resolution H3 index resolution
+   * @return the H3 index address
+   */
+  @ScalarFunction
+  public static long geoToH3(double longitude, double latitude, int resolution) {
+    return H3Utils.H3_CORE.geoToH3(latitude, longitude, resolution);
+  }
```

Comment:
We should also add the version with 2 arguments: `geoToH3(byte[] bytes, int resolution)`

Reply:
how does the signature match with same name?

Reply:
We match function name + argument count, so Pinot can treat them as different function